content/docs/started/getting-started-gke.md

```diff
@@ -23,6 +23,8 @@ brings the following advantages:
   * [Cloud Identity-Aware Proxy (Cloud IAP)](https://cloud.google.com/iap/) 
     makes it easy to securely connect to Jupyter and other
     web apps running as part of Kubeflow.
+  * Basic auth service supports simple username/password access to your 
+    Kubeflow. It is an alternative to Cloud IAP service.
```

Comment:
Can we add some guidance about basic auth.
Specifically can we make it clear that we recommend IAP for production, enterprise workloads. 
Basic auth should be used only for testing out Kubeflow. 
Furthermore it should only be used in projects without access to sensitive data.

Reply:
added.